README.md

```diff
@@ -1,5 +1,18 @@
 <p align="center"><img width=60% src="docs/media/ReactSDK-Logo.png">
 
+<table style="font-size: 14pt;">
+   <tr>
+   <th>Deprecation Notice</th>
+   </tr>
+   <tr><td>Enhancement and support for this version (<strong>release/8.6</strong>)
+      of the React SDK has ended with the release of Pega Infinity 8.8 in
+      October 2022.
+      <br>The <strong>main</strong> branch supports Pega Infinity 8.7 and 8.8
```

Comment:
Missing period at the end of line number 10. The content looks good otherwise.

Reply:
Fixed. Please re-review